### ISSUE TITLE
Pre-trained model for the ResNet-50 trained via AdvProp

### ISSUE THREAD

**Issue by anguyen8** [tingxueronghua/pytorch-classification-advprop#6]
Hi,

Thank you so much for your implementation!

It's exciting to see ResNet-50 trained via AdvProp reaches a similar accuracy to the vanilla ResNet-50! :) 

1. Would you be able to share the model that achieves 77.42% (as reported in README.md)?
2. Do you have any statistics on the **adversarial** accuracy of this model?

Thank you so much!

Anh

**Comment by tingxueronghua** [comment]
For 1, I would try, but I think it would be hard because I have left Alan's group as an intern. The weights were in the sever and I am not sure whether could I get it. Anyway, if you have prepared ImageNet, I think it is not hard to reproduce the result.

For 2, no, I did not test it.

**Comment by anguyen8** [comment]
Thanks for letting me know! I'll try to reproduce it.
